validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkFeedbackRequestValidatorTest.java

```diff
@@ -54,57 +55,65 @@
 import uk.gov.moj.sdt.domain.cache.api.ICacheable;
 import uk.gov.moj.sdt.validators.exception.InvalidBulkReferenceException;
 
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
 /**
  * Tests for {@link BulkCustomerValidatorTest}.
  *
  * @author d120520
  */
 
+@ExtendWith(MockitoExtension.class)
 public class BulkFeedbackRequestValidatorTest extends AbstractValidatorUnitTest {
     /**
      * Logger object.
      */
     private static final Logger LOGGER = LoggerFactory.getLogger(BulkFeedbackRequestValidatorTest.class);
 
     /**
-     * Subject for test.
+     * The Dao.
      */
-    private BulkFeedbackRequestValidator validator;
+    @Mock
+    private IBulkSubmissionDao mockIBulkSubmissionDao;
 
     /**
-     * The IBulkFeedbackRequest.
+     * Error messages cache.
      */
-    private IBulkFeedbackRequest bulkFeedbackRequest;
+    @Mock
+    private ICacheable errorMessagesCache;
 
     /**
-     * The Dao.
+     * Global Parameter cache.
      */
-    private IBulkSubmission bulkSubmission;
+    @Mock
+    private ICacheable globalParameterCache;
 
     /**
-     * The Dao.
+     * Subject for test.
      */
-    private IBulkSubmissionDao mockIBulkSubmissionDao;
+    private BulkFeedbackRequestValidator validator;
 
     /**
-     * The string reference for our bulk request.
+     * The IBulkFeedbackRequest.
      */
-    private String reference = " Bulk reference in request ";
+    private IBulkFeedbackRequest bulkFeedbackRequest;
 
     /**
-     * requestId.
+     * The Dao.

```

Comment:
This comment doesn't appear to be correct - I think it should be "Bulk Submission"